float32 vbus

int32 error0
uint8 state0
float32 vel0
float32 pos0
float32 curr0B
float32 curr0C
float32 temp0

int32 error1
uint8 state1
float32 vel1
float32 pos1
float32 curr1B
float32 curr1C
float32 temp1

